rest/client.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
-	clientVersion = "2.6.5"
+	clientVersion = "2.6.6"
```

Comment:
changelog should be updated alongside a version bump. I would suggest keeping these in a separate PR rather than in a feature PR

Reply:
Done.